content/OTP/OTPs_Explained.adoc

```diff
@@ -72,39 +72,38 @@ The YubiKey OTP generation is made up of the following fields, encrypted with a
 
-If the counter reaches the final value, the device can still be re-configured which would cause the counter to be reset.
-The field is stored in little-endian format, i.e. the least significant byte being stored first.
+The usage counter is stored in little-endian format, i.e. the least significant byte is stored first.
 
 === Timestamp
-The timestamp is a 24-bit field incremented with a rate of approximately 8 Hz. The timestamp value is set to a random value after startup from the internal random number generator.
+The timestamp is a 24-bit field incremented at a rate of approximately 8 Hz. At startup, it is set to a random value using the internal random number generator.
+
+The timestamp may be used by the verifying party to determine the time elapsed between two consecutive OTPs received during a single session.
 
-This field may be used by the verifying party to determine the time elapsed between two subsequent OTPs received during a session.
+This field automatically wraps from 0xffffff to 0. The verifying party must take this constraint into account. Given an 8 Hz rate, the timer will wrap approximately every 24 days.
 
-This field wraps from 0xffffff to 0 without any further action. If used by the verifying party, this condition must be taken into account. Given an 8 Hz rate, the timer will wrap approximately every 24 days. The field is stored in little-endian format, i.e. the least significant byte being stored first.
+The timestamp is stored in little-endian format.
 
 === Random number
 A 16-bit random number is picked from the internal random number generator to add some additional entropy to the final result.
 
 === Checksum
-A 16-bit ISO13239 1st complement checksum is added to the end. The checksum spans all bytes except the checksum itself. The checksum is
-verified by calculating the checksum of all bytes, including the checksum field. This shall give a fixed residual of 0xf0b8 if the checksum is valid. If the checksum is invalid, the OTP shall be rejected.
+A 16-bit, ISO13239, 1's-complement checksum is added at the end. The checksum spans all bytes except the checksum itself. It is verified by calculating the checksum of all bytes, including the checksum field, where a valid checksum produces a fixed residual of 0xf0b8. If the checksum is invalid, the OTP shall be rejected.
 
-The field is stored in little-endian format, i.e. the least significant byte being stored first.
+The checksum is stored in little-endian format (the least significant byte comes first).
```

Comment:
Let's be consistent with the previous paragraphs, no? :slightly_smiling_face: 

```suggestion
The checksum is stored in little-endian format, i.e. the least significant byte is stored first.
```

Reply:
Some times, I like to introduce variety into the mix.  :)
But sure, no problem!